CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
+# 5.5.0
+- Add support for /item/add_token/create endpoint
+
 # 5.4.0
-- Add support for Beta /item/add_token/create endpoint
+- Add item status for the investment product.
+- Add transaction_code field to the Transaction object.
```

Comment:
I believe this fixes issues with the changelog being out of sync

Reply:
yep, thanks for updating this!